app/src/main/java/chat/rocket/android/chatrooms/ui/ChatRoomsAdapter.kt

```diff
@@ -0,0 +1,87 @@
+package chat.rocket.android.chatrooms.ui
+
+import DateTimeHelper
+import DrawableHelper
+import android.content.Context
+import android.support.v7.widget.RecyclerView
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import chat.rocket.android.R
+import chat.rocket.android.helper.UrlHelper
+import chat.rocket.android.util.inflate
+import chat.rocket.android.util.setVisibility
+import chat.rocket.android.util.textContent
+import chat.rocket.common.model.BaseRoom.RoomType
+import chat.rocket.core.model.ChatRoom
+import com.facebook.drawee.view.SimpleDraweeView
+import kotlinx.android.synthetic.main.item_chat.view.*
+
+class ChatRoomsAdapter(private var dataSet: MutableList<ChatRoom>, private val context: Context) : RecyclerView.Adapter<ChatRoomsAdapter.ViewHolder>() {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder = ViewHolder(parent.inflate(R.layout.item_chat))
+
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
+        val chatRoom = dataSet[position]
+        val chatRoomName = chatRoom.name
+
+        holder.chatName.textContent = chatRoomName
+
+        if (chatRoom.type == RoomType.ONE_TO_ONE) {
+            // TODO Check the best way to get the current server url.
+            val canonicalUrl = chatRoom.client.restUrl.toString()
```

Comment:
The server URL is a setting called `Site_Url`.

Reply:
Thanks for notify @rafaelks 👍 
This is for both of us to know @luciofm ^^